clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -3952,6 +3981,16 @@ bool CompilerInvocation::ParseLangArgs(LangOptions &Opts, ArgList &Args,
     }
   }
 
+  if (const Arg *const A = Args.getLastArg(OPT_fcf_branch_label_scheme_EQ)) {
```

Comment:
```suggestion
  if (const Arg *A = Args.getLastArg(OPT_fcf_branch_label_scheme_EQ)) {
```

```suggestion
  if (const Arg *A = Args.getLastArg(OPT_fcf_branch_label_scheme_EQ)) {
```

For consistency with other site within this file.

Reply:
Fixed